addon/components/base/bs-modal/dialog.js

```diff
@@ -17,9 +18,10 @@ export default Component.extend({
   layout,
   classNames: ['modal'],
   classNameBindings: ['fade'],
-  attributeBindings: ['tabindex', 'style'],
+  attributeBindings: ['tabindex', 'style', 'aria-labelledby'],
   ariaRole: 'dialog',
   tabindex: '-1',
+  "aria-labelledby": alias('titleId'),
```

Comment:
I tend to not use `alias` when possible, as it introduces a two-way binding. Would prefer `readOnly` here.